plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -29,6 +29,7 @@
 
```

Comment:
What is the problem that you are fixing? Is this about concurrent access of ignite? Maybe we should run this test in serial way. WDYT?

Another idea is that we could potentially use `@Flaky` if the test can recover from the failure.

Reply:
> What is the problem that you are fixing? Is this about concurrent access of ignite? Maybe we should run this test in serial way. WDYT?

Yes, the concurrent request for the "add" or "drop" column. 

> Maybe we should run this test in serial way.

I feel some tasks is tend to test the concurrent scenario about the the metadata request, for those tests maybe we shouldn't do in serial way. For the tests that not tend to test concurrent scenario maybe we can. 
And could we add a behavior like `SUPPORT_CONCURRENT_ON_COLUMN_MODIFICATION`

@Flaky is marked as Deprecated

Reply:
> `@flaky` is marked as Deprecated

The point of this to discourage the usage. So people try to fix it. However here, if the issue is on the ignite side I think usage of that annotation is justified.

Reply:
> SUPPORT_CONCURRENT_ON_COLUMN_MODIFICATION

That sounds like an overkill to me.